### PR TITLE
Refactor save validation

### LINKED ISSUE #168: Push validated entries back to the DBs, overriding values if needed
Based on the user manipulation of the validation table, the db entries will need to be updated. At the very least, their validation status will change and some might be deactivated or have some of their values manually overridden.

### PR DESCRIPTION
**Pull request by dalonsoa**
Refactors the save validation code to run after the appropriate button is pressed in the validation page. This PR only concern the backend code. The integration with the front end will be done in #170, where we will do appropriate adjustments to both sides, as needed.

This code might be worth to run in a separate thread since it is sequential, row by row, and might take a while to complete. We will asses the performance after putting everything together. 

Close #168

### CODE REVIEW

**Review by tsmbland — APPROVED** [review]
Looks good. Just one comment

File: measurement/validation.py [hunk 1]
```diff
@@ -243,3 +243,43 @@ def generate_validation_report(
...
+    """Saves the validated data to the database.
+
+    Only the data that is flagged as "validate?" will be saved. Possible updated fields
+    are: value, maximum, minimum and is_active.
+
+    Args:
+        data: The dataframe with the validated data.
+    """
+    for _, row in data[data["validate?"]].iterrows():
+        current = Measurement.objects.get(id=row["id"])
```
[the remark this answers is not in the document]
Comment (tsmbland) [reply measurement/validation.py:258]: The way I'm currently doing it is to assign new rows with the ID of the last row in the table +1. This is how the javascript code did it, but is this what we want?

[the remark this answers is not in the document]
Comment (dalonsoa) [reply measurement/validation.py:258]: The id is created automatically by the database when creating a new entry, so we should not be creating them manually. In that sense, I think it was probably a mistake to allow that in the JS code, to start with.

[the remark this answers is not in the document]
Comment (dalonsoa) [reply measurement/validation.py:258]: Besides, what's the point of manually creating entries in the *validation* page? I can see a use case for manually creating new data, row by row, but the validation page is about that: validating data. It should allow for removing or tweaking outliers, but this is not the place for adding new data.

[the remark this answers is not in the document]
Comment (tsmbland) [reply measurement/validation.py:258]: > Besides, what's the point of manually creating entries in the _validation_ page? 

I agree entirely

**Pull request merged**